whois/whois.go

```diff
@@ -0,0 +1,53 @@
+package whois
+
+import (
+	"encoding/json"
+
+	model "github.com/owasp-amass/open-asset-model"
+)
+
+// WHOIS represents the WHOIS information of a domain.
+type WHOIS struct {
+	// Type represents the type of the WHOIS information.
+	Type string `json:"type"`
+
+	// CreatedDate represents the creation date of the domain or org associated with a netblock.
+	CreatedDate string `json:"created_date,omitempty"`
+
+	// LastModifiedDate represents the last modified date of the domain.
+	UpdatedDate string `json:"updated_date,omitempty"`
+
+	// ExpirationDate represents the expiration date of the domain.
+	ExpirationDate string `json:"expiration_date,omitempty"`
+
+	// DomainStatus represents the status of the domain.
+	DomainStatus []string `json:"domain_status,omitempty"`
+
+	// RegistryRegistrantID represents the registry registrant ID of the domain.
+	RegistryRegistrantID string `json:"registry_registrant_id,omitempty"`
+
+	// RegistryDomainID represents the registry domain ID of the domain.
+	RegistryDomainID string `json:"registry_domain_id,omitempty"`
```

Comment:
should this be billing? From the RAA Global RDAP Amendment spec, I see registrant, admin, technical, and billing: 

<img width="621" alt="image" src="https://github.com/owasp-amass/open-asset-model/assets/72798/79b36afa-2533-47a0-970d-bb5872c29d3f">

Reply:
These two variables should remain in the struct. But you are correct, I should add a **RegistryBillingID** in there

Reply:
**RegistryBillingID** has been added.